CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ Breaking changes:
         Adapter functions.
     * named `Base` because it's a Rails-ism.
     * It helps to isolate and highlight what the Adapter interface actually is.
+- [#1418](https://github.com/rails-api/active_model_serializers/pull/1418)
+  When generating a collection serialization's `json_key`, the `root:`
+  option is no longer pluralized. This enables you to generate any
```

Comment:
Maybe

> serialized collections now use the `root` option as is; now, only the root derived from the serializer or object is always pluralized.

(Makes me think we should allow this to be configured like in JSON API)